perception/lidar_apollo_segmentation_tvm/package.xml

```diff
@@ -21,6 +21,7 @@
   <depend>tf2_eigen</depend>
   <depend>tf2_geometry_msgs</depend>
   <depend>tf2_ros</depend>
+  <depend>tier4_autoware_utils</depend>
```

Comment:
This should be introduced in a separate PR, let me know if you want me to submit it.

Reply:
Yes, there is the PR for this changes https://github.com/autowarefoundation/autoware.universe/pull/4908

Reply:
I've merged #4908 now, thanks for the patch.